diff -urN mozjs-60.1.0.orig/python/mozbuild/mozbuild/virtualenv.py mozjs-60.1.0/python/mozbuild/mozbuild/virtualenv.py
--- mozjs-60.1.0.orig/python/mozbuild/mozbuild/virtualenv.py	2018-06-19 00:24:05.000000000 +0200
+++ mozjs-60.1.0/python/mozbuild/mozbuild/virtualenv.py	2019-01-27 14:30:42.181096929 +0100
@@ -66,8 +66,7 @@
     @property
     def virtualenv_script_path(self):
         """Path to virtualenv's own populator script."""
-        return os.path.join(self.topsrcdir, 'third_party', 'python',
-            'virtualenv', 'virtualenv.py')
+        return '/usr/bin/virtualenv-2'
 
     @property
     def bin_path(self):
@@ -424,7 +424,7 @@
                 handle_package(package)
 
             sitecustomize = os.path.join(
-                os.path.dirname(os.__file__), 'sitecustomize.py')
+                os.path.dirname(distutils.sysconfig.get_python_lib()), 'sitecustomize.py')
             with open(sitecustomize, 'w') as f:
                 f.write(
                     '# Importing mach_bootstrap has the side effect of\n'
